pkg/virt-launcher/virtwrap/network/podinterface_test.go

```diff
@@ -399,7 +399,11 @@ var _ = Describe("Pod Network", func() {
 				It("should populate MAC address", func() {
 					vmi := newVMIBridgeInterface("testnamespace", "testVmName")
 					vmi.Spec.Domain.Devices.Interfaces[0].MacAddress = "de-ad-00-00-be-af"
-					driver, err := podnic.getPhase1Binding(vmi, &vmi.Spec.Domain.Devices.Interfaces[0], &vmi.Spec.Networks[0], primaryPodInterfaceName)
+					podnic.vmi = vmi
+					podnic.iface = &vmi.Spec.Domain.Devices.Interfaces[0]
+					podnic.network = &vmi.Spec.Networks[0]
+					podnic.podInterfaceName = primaryPodInterfaceName
```

Comment:
Do you mind exporting this into a function ? 

This repeats 17 times in this file.

Something like the signature below would do.
```golang
func generatePodNic(vmi, iface, network, podIfaceName) PodNic {
    ....
}
```

Reply:
Done